lib/salus/package_utils/gradle_dependency_parser.rb

```diff
@@ -1,16 +1,11 @@
 module Gradle
-  GRADLE7 = "/opt/gradle/gradle-7.3.3/bin/gradle".freeze
-  GRADLE6 = "/opt/gradle/gradle-6.9.2/bin/gradle".freeze
-
   def gradle_dependencies
     dependency_metadata_regex = /-\s(?<group_id>.+):(?<artifact_id>.+):(?<version>.+)/
 
     # 'gradle dependencies' command needs to be run in the folder where buid.gradle is present.
-    shell_result = run_shell("#{GRADLE7} dependencies")
-
-    shell_result = run_shell("#{GRADLE6} dependencies") if !shell_result.success?
+    shell_result = run_shell("./gradlew dependencies")
     if !shell_result.success?
-      report_error("Gradle Version Not supported. Please Upgrade to gradle version 6 and above")
+      report_error(shell_result.stderr)
```

Comment:
Does `report_error` print to the txt output?

Reply:
yes, report error prints to the txt output

<img width="400" alt="Screen Shot 2022-06-08 at 2 20 53 PM" src="https://user-images.githubusercontent.com/23368832/172719520-f6516daf-1321-4ede-8dad-711e3434d985.png">